### PR TITLE
fix: bugs in traceloop integration

### PR DESCRIPTION
**Pull request by nirga**
- [X] Update import path to match current Traceloop version
- [X] Added a test for Traceloop <> LiteLLM integration
- [X] Logging for non-OpenAI calls (since LiteLLM doesn't use vendors standard libraries)

### CODE REVIEW

File: litellm/utils.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
     dataclass,
     field,
 )  # for storing API inputs, outputs, and metadata
+
```
Comment (ishaan-jaff) [inline litellm/utils.py]: it looks like there are a large amount of changes to this file. Can we avoid this ?

Comment (ishaan-jaff) [reply litellm/utils.py]: and it looks like they're all formatting changes

Comment (nirga) [reply litellm/utils.py]: VSCode auto-format on save 😅 
Done @ishaan-jaff

**Comment by ghost** [comment]
@nirga what env variables do we need on circle ci to pass traceloop testing?

**Comment by nirga** [comment]
@krrishdholakia 
For running the test -
ANTHROPIC_API_KEY
OPENAI_API_KEY
TRACELOOP_API_KEY (which you can get by signing up [app.traceloop.com](http://app.traceloop.com/))

**Pull request merged**